src/client/core/prevent-real-events.js

```diff
@@ -70,7 +70,11 @@ function preventRealEventHandler (e, dispatched, preventDefault, cancelHandlers,
     }
 }
 
-export default function preventRealEvents () {
+export function preventRealEvents () {
     listeners.initElementListening(window, PREVENTED_EVENTS);
```

Comment:
We can't do it because this method can be called twice for the window that will be recreated